interview-questions.md

```diff
@@ -9,43 +9,60 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What does calling super do in a class?
 
 Your answer:
-
+Super works with the the initialize command and holds the argument name. It is used to to borrow  the initialize method to assign an instance variable.
 Researched answer:
+Super calls a method to the parent class with the same name as the method super invoked. Some important things to know about super is that it can only be used inside a methed and it returns the result from the parent method. Also, it can be called multiple times.
 
-2. What is a gem?
+2. Whapt is a gem?
 
 Your answer:
+A gem is packaged ruby code that provide custom functionality.
```

Comment:
I would strongly recommend expanding on this answer.  For instance, talk about some of the gems that you have worked with, and what have you noticed about working with those gems.